snomed/com.b2international.snowowl.snomed.core.rest.tests/src/com/b2international/snowowl/snomed/core/rest/AllSnomedApiTests.java

```diff
@@ -112,6 +112,7 @@
 	// Performance test cases, should be the last tests to perform
 	SnomedConceptCreatePerformanceTest.class,
 	SnomedMergePerformanceTest.class,
+	SnomedExpressionLabelTest.class
```

Comment:
Please note the comment above. Performance benchmark tests should be the last tests to execute, please move this up in the execution order, could be next to the component API tests.

Reply:
Done